src/cloud/env/variables-post-deploy.md

```diff
@@ -131,6 +131,19 @@ Customize the list of pages used to preload the cache in the `post_deploy` stage
                - "store-page:/contact-us:code1|code2"
    ```
 
+### `WARM_UP_CONCURRENCY`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Configure _CONCURRENCY_ with which requests to _WARM_UP_PAGES_ will be sent. This configuration limits number of parallel connections. It can help to decrease server load in case of a big amount of pages to warm-up.
```

Comment:
```suggestion
Specify the number of concurrent requests to send during cache warmup operations to reduce server load. This value limits the number of parallel connections and is useful for environment configurations where the `WARM_UP_PAGES` post-deploy variable specifies a large number of pages for cache preloading.
```

Reply:
Any recommendation to what value to set?  The example shows "4".

Reply:
I didn't add a default value to do not potentially slowdown post-deploy process for customers who don't specify this value.
The number from example is a random one. It's difficult to provide the recommended value because it depends on customer's instance configuration.